src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -3358,10 +3358,10 @@ subroutine bilinear_shape_functions (X, Y, Phi, area)
 
   do qpoint=1,4
 
-    a = -X(1)*(1-yquad(qpoint)) + X(2)*(1-yquad(qpoint)) - X(3)*yquad(qpoint) + X(4)*yquad(qpoint) ! d(x)/d(x*)
```

Comment:
The changes on these lines are mathematically equivalent to what was here before, but in specifying the correct order of arithmetic to give rotational symmetry, they are likely to change answers for any cases that use a dynamic ice sheet.  That being said, I do agree that these precise changes are both correct an necessary.

Reply:
I do not believe that there are any widely used cases with a dynamic ice sheet that would be adversely impacted by these specific changes, so I do not think that we need to add an answer date flag here to recover the previous (indeterminate) order of arithmetic.